modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js

```diff
@@ -0,0 +1,255 @@
+}
+
+function nullOrValue(value) {
+    if (value === undefined) {
+        return null;
+    }
+
+    return value;
+}
+
+/**
+ * @typedef ContentType
+ * @type Object
+ * @property {string} name Name of the content type.
+ * @property {string} displayName Display name of the content type.
+ * @property {string} description Description of the content type.
+ * @property {string} superType Name of the super type, or null if it has no super type.
+ * @property {boolean} abstract Whether or not content of this type may be instantiated.
+ * @property {boolean} final Whether or not it may be used as super type of other content types.
+ * @property {boolean} allowChildContent Whether or not allow creating child items on content of this type.
+ * @property {string} displayNameExpression ES6 string template for generating the content name based on values in the content form.
+ * @property {object} [icon] Icon of the content type.
+ * @property {object} [icon.data] Stream with the binary data for the icon.
+ * @property {string} [icon.mimeType] Mime type of the icon image.
+ * @property {string} [icon.modifiedTime] Modified time of the icon. May be used for caching.
+ * @property {object[]} form Form schema represented as an array of form items: Input, ItemSet, Layout, OptionSet.
+ */
+
+/**
+ * This function fetches a content.
+ *
+ * @example-ref examples/content/get.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} [params.versionId] Version Id of the content.
+ *
+ * @returns {object} The content (as JSON) fetched from the repository.
+ */
+exports.get = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetContentHandler');
+    bean.key = required(params, 'key');
+    bean.versionId = nullOrValue(params.versionId);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns a content attachments.
+ *
+ * @example-ref examples/content/getAttachments.js
+ *
+ * @param {string} key Path or id to the content.
+ *
+ * @returns {object} An object with all the attachments that belong to the content, where the key is the attachment name. Or null if the content cannot be found.
+ */
+exports.getAttachments = function (key) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentsHandler');
+    bean.key = nullOrValue(key);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns a data-stream for the specified content attachment.
+ *
+ * @example-ref examples/content/getAttachmentStream.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.name Attachment name.
+ *
+ * @returns {*} Stream of the attachment data.
+ */
+exports.getAttachmentStream = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentStreamHandler');
```

Comment:
Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)